<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrameConfig</class>
 <widget class="QWidget" name="FrameConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Frame Sizes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string comment="The name of the new item">Name:</string>
          </property>
          <property name="buddy">
           <cstring>sizeEdit</cstring>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="KLineEdit" name="sizeEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="spacing">
         <number>-1</number>
        </property>
        <item row="0" column="0" rowspan="5">
         <widget class="QTreeWidget" name="sizeTree">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string comment="The name of the item">Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string comment="the width of the frame">Width</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string comment="the height of the frame">Height</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addButton">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="upButton">
          <property name="text">
           <string>Move Up</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="downButton">
          <property name="text">
           <string>Move Down</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
